README-hi.md

```diff
@@ -0,0 +1,27 @@
+
+
+## नए आयोजकों के लिए
+* **हमारे जिटर चैट में शामिल हों:** पूरे विश्व में अध्ययन समूह हैं। हम संसाधनों और विचारों को जोड़ने और साझा करने के लिए एक ऑनलाइन [गिटार चैट](https://gitter.im/mozillascience/studyGroup) का उपयोग करते हैं (यदि आप GitHub पर सेट नहीं हैं, तो आप GitHub के साथ प्रवेश कर सकते हैं, या ट्विटर आईडी का उपयोग कर सकते हैं बस अभी तक)। अगर आप नमस्ते कहना चाहते हैं, तो कृपया अपने आप को चैट में शामिल करें, हमें बताएं कि आप कहां हैं, और आप अपने नए अध्ययन समूह के बारे में क्या सोच रहे हैं या योजना बना रहे हैं हम आपको मिलाने के लिए उत्सुक हैं
+* **अध्ययन समूह की ओरिएंटेशन गाइड देखें:** अगर आपको लगता है कि आप अपना स्वयं का समूह शुरू कर सकते हैं, [स्टडी ग्रुप ओरिएन्टेशन गाइड](https://mozillascience.github.io/study-group-orientation/index.html) समूह की बैठकों को चलाने, आपकी स्टडी ग्रुप की वेबसाइट का सुपर आसान सेटअप, गिटहब का उपयोग करके ऑनलाइन सहयोग कर रहा है, अनुसंधान अभ्यास को खोलने के लिए एक परिचय, और अधिक। [यहां अनुभाग के बारे में](https://mozillascience.github.io/study-group-orientation/1-about-study-groups.html) पर एक नज़र डालें और [स्टडी ग्रुप लीड रोल](https://mozillascience.github.io/study-group-orientation/1.1-lead-role.html) पर अनुभाग - इन्हें आपको यह पता लगाने में सहायता करनी चाहिए कि क्या आप समूह शुरू करना चाहते हैं।
+* **एक ऑनलाइन ओरिएंटेशन मीटिंग में आओ:** हम एक ऑनलाइन ओरिएंटेशन चल रहे होंगे - 4 मीटिंग्स की एक श्रृंखला, - अगले महीने (नवंबर 2016!) और 2017 के जनवरी में नए समूह की बिक्री के लिए लीड्स। दूसरे समूह के साथ मिलकर, नेटवर्क, और विचार साझा करने का एक बड़ा अवसर। यदि आप हमसे जुड़ने में रुचि रखते हैं, [हमें यह फ़ॉर्म भरकर बताएँ।](https://docs.google.com/forms/d/e/1FAIpQLSdtKqAMQnKri-0xLx4hD_fpb000n9czsQd4oo9B2JUgtuIVlg/viewform?c=0&w=1)
+* **आचार संहिता पढ़ें:** यह अध्ययन समूह कार्यक्रम सभी के लिए है - हम [नियमों के सेट](https://www.mozillascience.org/code-of-conduct/) का पालन करते हैं| जो सभी को आवश्यकता होती है सम्मान के साथ इलाज किया हमें ऐसा स्थान बनाने में मदद करें जहां हर कोई स्वागत करता है, और हम सभी को बेहतर समय मिलेगा!
+* **इस रेपो को देखें:** शीर्ष दाईं ओर, एक बटन है जो 'वॉच' कहता है; इसे क्लिक करें, और खुद को 'देखना' पर सेट करें इससे आपको नई चर्चाओं की ईमेल सूचनाएं भेजी जाएंगी; अगर आप ईमेल नहीं चाहते हैं, लेकिन केवल GitHub पर एक चेतावनी चाहें, तो सेटिंग्स सेटिंग अधिसूचना केंद्र में सेटिंग्स को बदलें (सेटिंग्स शीर्ष दाहिनी ओर छोटी सी कॉग है)
+ 
+## अपनी खुद की मोज़िला अध्ययन समूह वेबसाइट कैसे सेट करें
+
+घटनाओं के आयोजन के लिए अपनी खुद की सुंदर मोज़िला अध्ययन समूह की वेबसाइट (यह इस तरह दिखती है!) को स्थापित करने की आवश्यकता है [ठीक यहां ओरिएंटेशन गाइड में है।](https://mozillascience.github.io/study-group-orientation/3.3-get-online.html) यदि आप गिटहब के लिए नए हैं, चिंता न करें, [यहां इसकी एक परिचय है](https://mozillascience.github.io/study-group-orientation/3.1-collab-vers-github.html) एक नज़र डालें, और अगर आपको किसी भी मदद की आवश्यकता है, तो आप [गिटार चैट](https://gitter.im/mozillascience/studyGroup) में एक प्रश्न पूछ सकते हैं या ईमेल sciencelab@mozillafoundation.org।
```

Comment:
@AnamikaD   
Please replace घटनाओं with अध्ययन समूह, because घटनाओं doesn't make any sense to organize.